### PR TITLE
clarification on when to use ma=false/true #1696 

### PR DESCRIPTION
**Pull request by aristotelis-economides**
fixed `multianimal:False` on the [User Overview Guide](https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/UseOverviewGuide.md)

From this issue: https://github.com/DeepLabCut/DeepLabCut/issues/1696

### CODE REVIEW

File: docs/UseOverviewGuide.md [hunk 1]
```diff
@@ -80,15 +80,12 @@ We highly recommend using 2.2 first in the Project Manager GUI ([Option 3](docs/
 ##### *What scenario do you have?*
 
 - **I have single animal videos, but I want to use the advanced tracking features & updated network capabilities introduced (for multi-animal projects) in DLC2.2:**
-   - quick start: when you `create_new_project` just set the flag `multianimal=True`.
+   - quick start: when you `create_new_project` just set the flag `multianimal=False`. You still get lots of upgrades!
```
Comment (MMathisLab) [inline docs/UseOverviewGuide.md]: this is not correct; it is written correctly. You can use PAFs with single animals, but MA must be set to TRUE.

**Review by MMathisLab — DISMISSED** [review]

**Comment by aristotelis-economides** [comment]
@MMathisLab  oh I see, I was confused. I re-typed it so it is a bit more explicit, let me know if that works

**Comment by MMathisLab** [comment]
most users will not know what this means; I prefer to leave it as it was ;)

**Comment by MMathisLab** [comment]
well, you can leave open as to fix the typo ;)

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**